packages/eslint-config/src/config/javascript.ts

```diff
@@ -28,7 +28,6 @@ const config: TSESLint.FlatConfig.ConfigArray = [
 if (!isInEditorEnv()) {
   config.push({
     rules: {
-      'no-unused-vars': 'off',
```

Comment:
According to the usage of [eslint-plugin-unused-imports](https://github.com/sweepline/eslint-plugin-unused-imports#usage), may we disable `no-unused-vars`/`@typescript-eslint/no-unused-vars ` and use `unused-imports/no-unused-vars` instead?